build.gradle

```diff
@@ -16,15 +17,23 @@ subprojects {
     }
 
     dependencies {
-        implementation group: 'org.springframework.boot', name: 'spring-boot-starter-web', version: '2.7.6'
-        compileOnly group: 'org.projectlombok', name: 'lombok', version: '1.18.24'
-        testImplementation 'org.junit.jupiter:junit-jupiter-api:5.9.0'
-        testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.9.0'
+        compileOnly 'org.projectlombok:lombok'
+        annotationProcessor 'org.projectlombok:lombok'
+        testCompileOnly 'org.projectlombok:lombok'
+        testAnnotationProcessor 'org.projectlombok:lombok'
+        testImplementation 'org.junit.jupiter:junit-jupiter-api'
+        testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
     }
 
     dependencyManagement {
         imports {
-            mavenBom("org.springframework.boot:spring-boot-dependencies:2.7.6")
+            mavenBom("org.springframework.boot:spring-boot-dependencies:2.7.1")
```

Comment:
move versions to the properties 

```
buildscript {
    ext {
        set 'springBootVersion', '2.7.5'
         }
 
}
```

and then reference to this vars from all places. i.e.

`mavenBom("org.springframework.boot:spring-boot-dependencies:${springBootVersion}")`

so then you can simply switch version without scanning whole dependencies.